packages/radio/src/RadioCards/RadioCards.tsx

```diff
@@ -8,7 +8,8 @@ import React from "react";
 
 import { baseStyles } from "./RadioCards.css";
 
-type RootProps = React.ComponentPropsWithoutRef<typeof RadioGroup.Root>;
+type RootProps = React.ComponentPropsWithoutRef<typeof RadioGroup.Root> &
+  Omit<TgphComponentProps<typeof Stack>, "tgphRef">;
```

Comment:
We should omit `Stack`’s `tgphRef` prop here because the `Root` component in this file uses `forwardRef` — correct?

Reply:
No I think we'd want to keep it because tgphRef eventually ends up down to Stack which is where it's useful. We actually need to remove forwardRef from here entirely.